### PR TITLE
Updated docs and copyright

### PR DESCRIPTION
**Pull request by bpoplauschi**
- Updated `README.md` **Migrating to 2.x** section with more details.
- Updated documentation files and added them as a folder inside the framework workspace
- Updated copyright years

### CODE REVIEW

File: Documentation/CustomFormatters.md [hunk 1]
```diff
@@ -154,41 +162,36 @@ MyCustomFormatter.m
...
+        case DDLogFlagInfo  : logLevel = @"I"; break;
+        case DDLogFlagDebug : logLevel = @"D"; break;
         default             : logLevel = @"V"; break;
     }
-    
-    NSString *dateAndTime = [threadUnsafeDateFormatter stringFromDate:(logMessage->timestamp)];
-    NSString *logMsg = logMessage->logMsg;
+
+    NSString *dateAndTime = [threadUnsafeDateFormatter stringFromDate:(logMessage.timestamp)];
+    NSString *logMsg = logMessage.message;
```
Comment (rivera-ernesto) [inline Documentation/CustomFormatters.md]: I think that for performance reasons direct ivar access was preferred in heavily called methods like this one.

File: Classes/CocoaLumberjack.swift [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Software License Agreement (BSD License)
 //
-// Copyright (c) 2014, Deusty, LLC
+// Copyright (c) 2014-2015, Deusty, LLC
```
Comment (rivera-ernesto) [inline Classes/CocoaLumberjack.swift]: As for updating the copyright, I wonder if "Deusty, LLC" is still pertinent now that it is part of a separate open source organization. @robbiehanson ?

**Comment by rivera-ernesto** [comment]
Thanks for the work!

**Comment by bpoplauschi** [comment]
Until we get a response from @robbiehanson, we should probably merge this and release 2.0 as it is. From my point of view, I only want to setup the unit testing architecture in case that requires some other breaking changes.

**Comment by bpoplauschi** [comment]
One further note, seems CocoaPods is coming close to a release of 0.36, since they released a rc 9 days ago.

**Pull request merged**